grid2op/Agent/TopologyGreedy.py

```diff
@@ -28,7 +28,8 @@ def __init__(self, action_space):
     def _get_tested_action(self, observation):
         if self.li_actions is None:
             res = [self.action_space({})]  # add the do nothing
-            res += self.action_space.get_all_unitary_topologies_change(self.action_space)
+            # better change are still "bugged" for now
```

Comment:
What do you mean ?

Reply:
I am not 100% sure the "change" are all computed. Maybe we are still missing some, so better, in my opinion to have agent that "set" the stuff for the Greedy agent.